core/src/main/java/org/apache/gravitino/listener/api/EventListenerPlugin.java

```diff
@@ -95,17 +97,29 @@ enum Mode {
   void stop() throws RuntimeException;
 
   /**
-   * Handles events generated after the completion of an operation. Implementers are responsible for
-   * processing these events, which may involve additional logic to respond to the operation
-   * outcomes.
+   * Handles events generated after the completion of an operation.
    *
-   * <p>This method provides a hook for post-operation event processing, allowing plugins to react
-   * or adapt based on the event details.
+   * <p>This method provides a hook for post-operation event processing, you couldn't change the
+   * resource in the event.
    *
-   * @param event The event to be processed.
-   * @throws RuntimeException Indicates issues encountered during event processing.
+   * @param postEvent The post event to be processed.
+   * @throws RuntimeException Indicates issues encountered during event processing, this has no
+   *     affect to the operation.
    */
-  void onPostEvent(Event event) throws RuntimeException;
+  default void onPostEvent(Event postEvent) throws RuntimeException {}
```

Comment:
Why do we change to `default`?

Reply:
after supporting pre event,  the user who only interest on pre event, could only implement onPreEvent interface, no need to implement onPostEvent interface